src/main/scala/nl.knaw.dans.easy.springfield/Command.scala

```diff
@@ -58,10 +58,10 @@ object Command extends App
   val result: Try[FeedBackMessage] = opts.subcommand match {
     case Some(cmd @ opts.listPresentations) =>
       debug("Calling list-presentations")
-      getPresentationList(cmd.user()).map(_.mkString(", "))
+      getPresentationList(cmd.user()).map(_.map(_.toString().replaceAll("[()]", "")).mkString("\n", "\n", ""))
     case Some(cmd @ opts.listFiles) =>
       debug("Calling list-files")
-      getFileList(cmd.user()).map(_.mkString(", "))
+      getFileList(cmd.user()).map(_.map(_.toString().replaceAll("[()]", "")).mkString("\n", "\n", ""))
```

Comment:
Note that the first output line is not a header line but `OK:` due to
https://github.com/DANS-KNAW/easy-springfield/blob/4b68d2a33e5e0ef15ccebc21ba3d48992a028701/src/main/scala/nl.knaw.dans.easy.springfield/Command.scala#L204
Maybe you want an argument for an output file?

Reply:
The OK-message could repeat the output file name (absolute path?) and/or number of records.